reference/shipping.v3.yml

```diff
@@ -382,8 +382,8 @@ components:
       title: customsInformation
       description: Data about the customs information object.
       type: object
-      minProperties: 5
-      maxProperties: 5
+      minProperties: 7
+      maxProperties: 7
```

Comment:
I'm not familiar with this - what does this describe? That all fields are required?

Reply:
Yeah, I think it was meant to describe that the fields are required (not sure why it was documented like this before). I marked the fields as required in `customsInformation_request` .